src/main/java/frc/robot/Robot.java

```diff
@@ -215,6 +242,29 @@ public CommandBase camToggle() {
         return camTogglecmd;
     }
 
+    public CommandBase enableTargeting() {
+        CommandBase cmd = new InstantCommand(() -> {
+            SmartDashboard.putBoolean("Is Shooting", true);
```

Comment:
If we use this variable a bunch, maybe use [getEntry](https://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/smartdashboard/SmartDashboard.html#getEntry(java.lang.String)) and the entry's `getBoolean` and `setBoolean`